package.json

```
@@ -7,13 +7,13 @@
     "node": ">=6.0.0"
   },
   "scripts": {
-    "deps": "python3 -m pip install -t lib -r requirements.txt --upgrade",
+    "deps": "python3 -m pip install -t lib -r requirements.txt --upgrade; cd py2; python3 -m pip install -t lib -r requirements.txt --upgrade",
```

```suggestion
    "deps": "python3 -m pip install -t lib -r requirements.txt --upgrade; cd py2; python2 -m pip install -t lib -r requirements.txt --upgrade",
```
Should this be python2 or I guess python3 has the backward compatibility to install py2 deps. I think the reason we can use python3 hers is because they are no overlapping dependencies between py2 and py3 services?

---

Inspired by this comment, I have a workaround for https://github.com/GoogleChrome/chromium-dashboard/pull/1653. I will install all dependencies in py3 but run in py2. WDYT? (Sorry for going back and forth on this discussion)

---

I was thinking of keeping it as python3 so that we are able to run `npm run deps` even on a workstation that has no python2 installed.  It seems to work.